zhaquirks/schneiderelectric/thermostat.py

```diff
@@ -572,5 +579,213 @@ class AttributeDefs(CustomCluster.AttributeDefs):
     .replaces(SEMetering, endpoint_id=5)
     .replaces(SECycleTime)
     .replaces(SEHeatingCoolingOutput)
+    .number(
+        cluster_id=SEMetering.cluster_id,
+        endpoint_id=5,
+        attribute_name=SEMetering.AttributeDefs.se_fixed_load_demand.name,
+        translation_key="fixed_load_demand",
+        fallback_name="Fixed Load Demand",
```

Comment:
From [HA Dev Docs](https://developers.home-assistant.io/docs/core/entity/#:~:text=Entity%20names%20should%20start%20with%20a%20capital%20letter%2C%20the%20rest%20of%20the%20words%20are%20lower%20case): "Entity names should start with a capital letter, the rest of the words are lower case".

So, this should be "Fixed load demand". All `fallback_name`s need to be changed to match that style, as we'll use those names to add to the zha `strings.json` for the default English name.

Reply:
Good point, fixed it :+1: